src/pages/Review.jsx

```diff
@@ -17,6 +17,10 @@ export default function Review() {
             .then(({ data }) => setReviews(data.reviews))
     }, []);
 
+    if (!reviews.length) {
+      return <p>Loading...</p>    
+    }
```

Comment:
This is not the way we have shown you how to implement loading patterns.
Read these notes if you're not sure how to do it: https://notes.northcoders.com/courses/js-front-end/useEffect-data-fetching